packages/core/react-loosely-lazy/src/utils/index.ts

```diff
@@ -10,12 +10,31 @@ export const displayNameFromId = (id: string) => {
 
 /**
  * Checks to see if we are running inside a node environment or not.
- * Covers jsdom environments.
+ * Covers jsdom environments and other environments which simply remove
+ * window.
  *
  * @see https://github.com/jsdom/jsdom/issues/1537
  */
 export const isNodeEnvironment = () => {
-  return globalThis !== globalThis.window;
+  // Some SSR environments remove the window object
+  if (typeof window === 'undefined') {
+    return true;
+  }
+
+  // Official way to check for JSDOM
```

Comment:
Is that documented somewhere? Also we should treat jsdom as a browser, and return false